src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useEffect } from 'react';
+import { withAuth } from '@okta/okta-react';
+import { useDispatch, useSelector } from 'react-redux';
+import { RouteComponentProps } from 'react-router-dom';
+import { DateTime } from 'luxon';
+import {
+  DescriptionList,
+  DescriptionTerm,
+  DescriptionDefinition
+} from 'components/shared/DescriptionGroup';
+import convertBoolToYesNo from 'utils/convertBoolToYesNo';
+import Header from 'components/Header';
+import { getSystemIntake } from '../../actions/systemIntakeActions';
+import { AppState } from '../../reducers/rootReducer';
+
+export type SystemIDRouterProps = {
+  systemID: string;
+};
+
+export type GRTSystemIntakeReviewProps = RouteComponentProps<
+  SystemIDRouterProps
+> & {
+  auth: any;
+};
+
+export const GRTSystemIntakeReview = ({
+  match,
+  auth
+}: GRTSystemIntakeReviewProps) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const fetchSystemIntake = async (): Promise<void> => {
+      dispatch(
+        getSystemIntake(await auth.getAccessToken(), match.params.systemID)
+      );
+    };
+    fetchSystemIntake();
+  }, [auth, dispatch, match.params.systemID]);
+  const systemIntake = useSelector(
+    (state: AppState) => state.systemIntake.systemIntake
+  );
+  const fundingDefinition = () => {
+    const isFunded = convertBoolToYesNo(!!systemIntake.fundingSource);
+    if (systemIntake.funding_source) {
+      return `${isFunded}, ${systemIntake.fundingSource}`;
+    }
+    return isFunded;
+  };
+  const issoDefinition = () => {
+    const hasIsso = convertBoolToYesNo(!!systemIntake.isso);
+    if (systemIntake.isso) {
+      return `${hasIsso}, ${systemIntake.isso}`;
+    }
+    return hasIsso;
+  };
+
+  return (
+    <div className="system-intake__review margin-bottom-7">
+      <Header />
+      <div className="grid-container">
+        <h1 className="font-heading-xl margin-top-4">CMS Sytem Request</h1>
+        {!systemIntake && (
+          <h2 className="font-heading-xl">
+            System intake with ID: {match.params.systemID} was not found
+          </h2>
+        )}
+        {systemIntake && (
```

Comment:
Correct me if I'm wrong, but the system intake form field content looks like the same as what is on the review page when filling out the system intake.

We can pull out that common stuff into a component and reuse it in the reviewer and requester track.

Reply:
Ah I think I see what's going on. We're going to need to either:

1. Match our Redux store model to the API model
2. Convert the API model to the Redux Formik model before storing it in Redux.

Having this consistency will let us reuse the existing markup in the system intake review page.

Reply:
Yep, that's why i did it separately, plus there are some parts that we infer from the API model, like collaborators or ISSO

Reply:
I'll make a follow up pr after this is merged to do that refactor.

Reply:
@christopherhuii just want to know if this is okay to delay until after the functionality gets in, but still prioritize that refactor (and not push off forever)?

Reply:
That's fine with me. We need to decide how we're going to handle the model between frontend and backend before we fetch saved system intakes for a user to finish filling out.